package.json

```diff
@@ -96,8 +95,6 @@
     "react-useportal": "1.0.17"
   },
   "resolutions": {
-    "@types/react": "17.0.48",
-    "@types/react-dom": "17.0.17",
     "postcss": "^8.3.11"
   },
   "peerDependencies": {
```

Comment:
I think we probably should add `"@types/react"` here as well.

Reply:
@huwshimi  As a peer dependency? Why would it be needed if we have it as dependency?

Reply:
We have these peer dependencies:
```
"react": "^17.0.2 || ^18.0.0",
"react-dom": "^17.0.2 || ^18.0.0",
```
So adding `"@types/react": "^17.0.2 || ^18.0.0"` should mean that the correct types get resolved for the correct React version.

Reply:
Ah, yes, good call.

Reply:
@huwshimi If the react-components now use React 18, is it correct to have 17 in peer dependencies? Would it work with React 17, or should we have React 18 only in peer dependencies now?

Reply:
I've just quickly tested it and react-components worked with a project using React 17. I think we want to have React 17 in peer deps for now otherwise those projects will get incorrect peer dependency warnings.